rpc_jobs/periodic_cleanup.yml

```diff
@@ -0,0 +1,57 @@
+- job:
+    name: 'Periodic-Cleanup'
+    project-type: workflow
+    parameters:
+      - string:
+          name: "INSTANCE_AGE_LIMIT"
+          default: "48"
+          description: |
+            Hours. Instances older than this will be removed.
+      - string:
+          name: "INSTANCE_PREFIX"
+          default: "ra"
+          description: |
+            Only instances whose names match the supplied prefix will be cleaned
+            up.
+      - string:
+          name: "OS_REGION_NAME"
```

Comment:
We call this REGION everywhere else.   You could standardise, but I'm not concerned enough to want to block the review.